src/Avalonia.Native/TrayIconImpl.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using Avalonia.Controls.Platform;
+using Avalonia.Native.Interop;
+using Avalonia.Platform;
+
+#nullable enable
+
+namespace Avalonia.Native
+{
+    class TrayIconEvents : CallbackBase, IAvnTrayIconEvents
+    {
+        private TrayIconImpl _parent;
```

Comment:
this field is not used anywhere, do we need it?

Reply:
events class not needed removed.